### DISCUSSION TITLE
"there was an issue with btcpay server. please click on "proceed to btcpay" again."

### DISCUSSION THREAD

**Discussion by DarkPID** [btcpayserver/btcpayserver#4234] [Technical Support]
Hi there

Just recently, users on my website have been getting an issue when attempting to check out with btcpay. 
The error message is- "there was an issue with btcpay server. please click on "proceed to btcpay" again."
Does anyone have any idea how to fix the issue? Trying to check out again gives the same error.

![Screenshot from 2022-10-24 17-50-31](https://user-images.githubusercontent.com/109679025/197657101-ae77cb8e-0c2b-434e-979e-ff393b5aa609.png)

**Comment by ndeet** [comment]
Hi, seems you use WooCommerce with the BTCPay Legacy Plugin. In the plugin settings you should be able to enable a "debug log" and post the result here after you attemtped an payment. There may be also some useful error logs in the webservers (Apache, Nginx) error log (location depends on how you host your woocommerce store).
